addons/rating/models/rating_mixin.py

```diff
@@ -1,10 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
as discussed on the task, I would make it inherit from mail.thread (and so you can remove the rating_ids field here)

Reply:
Indeed, that’s more natural but it has introduced a problem in the sale_subscription addons solved by introducing a new mixin, portal.rating.mixin that inherits from portal and rating mixin. It is used as base class for sale.order in sale_subscription to solve the inheritance problem (portal mixin was not called anymore causing problem, adding explicitly portal.mixin as base class didn’t solve the problem).